### ISSUE TITLE
Bug in returning "_sample" and "_sample_dict" in fastAPI endpoint

### ISSUE THREAD

**Issue by khoroshevskyi** [pepkit/pephub#124]
I am trying to write get endpoint that returns dictionary (or pydatic object) like this:
```
{
  "project": {
    "description": "This is project description",
    "name": "hello",
    "_config": {
      "pep_version": "2.0.0",
      "sample_table": "/tmp/tmpv6yivjij/sample_table.csv",
      "sample_modifiers": {
        "append": {
          "new_append": "dddd"
        },
      }
    },
    "_subsample_dict": None,
    "_sample_dict": {'organism': {'0': 'pig', '1': 'pig', '2': 'frog', '3': 'frog'}, 'sample_name': {'0': 'pig_0h', '1': 'pig_1h', '2': 'frog_0h', '3': 'frog_1h'}},
    "_sample1": {"a": 1, "b":2},
    "_something_else": {"different":"dict"}
  }
        }
```
Response body:
```
{
  "project": {
    "description": "This is project description",
    "name": "hello",
    "_config": {
      "pep_version": "2.0.0",
      "sample_table": "/tmp/tmpv6yivjij/sample_table.csv",
      "sample_modifiers": {
        "append": {
          "new_append": "dddd"
        }
      }
    },
    "_subsample_dict": null,
    "_something_else": {
      "different": "dict"
    }
  }
}
```

But i ran into a problem:
All keys with values are returned (after get request), except "_sample_dict" and "_sample1".

I found that this issue can lead to pydantic: https://github.com/pydantic/pydantic/issues/288 , but it seams that is not a problem as endpoint request returns keys with underscore: `_config` and `_something_else` and `_subsample_dict`.

Does someone know where is a problem?

@nleroy917 @nsheff

**Comment by khoroshevskyi** [comment]
I found issue: key can't start with `_sa`:
https://github.com/tiangolo/fastapi/blob/d666ccb62216e45ca78643b52c235ba0d2c53986/fastapi/encoders.py#L104

This issue is described here:
https://github.com/tiangolo/fastapi/pull/1987 and https://github.com/tiangolo/fastapi/pull/1864

**Comment by nleroy917** [comment]
That's interesting... I wonder if it has anything to do with the fact that when you convert a SQLAlchemy model into a dictionary with `__dict__` you are left with `_sa_instance_state`: https://stackoverflow.com/a/10370224/13175187

And we unfortunately have keys that start with that. Can we just, for now on the server, remove all `_sa`?

**Comment by nleroy917** [comment]
@Khoroshevskyi whats the status of this issue?

**Comment by khoroshevskyi** [comment]
This issue was fixed in pephub 0.7.0 release. (Pephub returns dict with keys without underscore)
